manifests/wego-app/helm_watcher_role.yaml.tpl

```diff
@@ -0,0 +1,31 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: helm-watcher-role
```

Comment:
is this too ambiguous? Since its not namespaced maybe worth putting `wego-` prefix? Idk I don't feel that strongly but eh :shrug: